katsdpimageutils/primary_beam_correction.py

```diff
@@ -96,54 +121,91 @@ def central_freq(path):
     return np.array(c_freq_plane)
 
 
-def cosine_power_pattern(separation_rad, c_freq):
+def check_band_type(path):
+    """Check band type using information from the FITS header and return appropriate
+       katbeam model name.
+
+    Parameters
+    ----------
+    path : str
+        FITS file
+
+    Returns
+    -------
+    output_file : str
+        katbeam model name
+    """
+    raw_image = read_fits(path)
+    band = raw_image.header.get('BANDCODE')
+    if band is None:
+        logging.info('----------------------------------------')
```

Comment:
Change this to `logging.warning(...)` (or remove it entirely) since if the logging level is `info` the logger will just print this and not the warning.